src/code/z_effect_soft_sprite_old_init.c

```diff
@@ -745,12 +771,12 @@ void EffectSsFhgFlash_SpawnLightBall(PlayState* play, Vec3f* pos, Vec3f* velocit
 }
 
 /**
- * Spawn a shock effect
+ * Spawn a purple shock/lighting effect.
  *
- * param determines where the ligntning should go
- * 0: don't attach to any actor. spawns at the position specified by pos
- * 1: spawn at one of Player's body parts, chosen at random
- * 2: spawn at one of Phantom Ganon's body parts, chosen at random
+ * @param actor If param is `FHGFLASH_SHOCK_PG`, the Phantom Ganon actor. Unused otherwise.
+ * @param pos If param is `FHGFLASH_SHOCK_NO_ACTOR`, the position of the effect. Unused otherwise.
+ * @param scale The effect will be `20*rand(1..2)*scale/100` units wide.
```

Comment:
its probably fine to omit the specifics on how the scale is calculated here, it can be found in the code.
can maybe say "the base scale to which some randomness is applied", or something like that

Reply:
yeah I guess the rand() part can be omitted,
but wouldn't it be useful to have a basic idea of how many units a scale value means?
I don't know how much it varies across effects, but if there are effects for which the scale should be 0.1 and others 10 to have a 10 units wide particle, browsing the code would get annoying fast imo

(and trying random numbers is cool but we can do better <img src="https://cdn.discordapp.com/emojis/943102210366119956.webp?size=56&quality=lossless" alt="skawoUHHUH" height="24">)

Reply:
(and the final size is not just about the code, it's the assets too)

Reply:
if it can be conveyed in a more digestible format sure, but id rather not be writing code in comments.
The effects do go for a consistent scaling with / 100 being very common

while I do agree with the sentiment, even with the information provided in the comment, I still dont think itd be easy to formulate what the scale should be. looking at existing usages and trying numbers is probably always going to be the best way lol

Reply:
Maybe something like "A scale of 50 makes the effect around 10 units wide" ? Or some other wording that suggests a "typical" scale

Very simply: `@param scale e.g. pass 50 for 10 units` ?

Reply:
I guess id be fine with "20 * scale / 100 units wide, with some randomness applied", or something